### ISSUE TITLE
uncompatible and not portable, (not postgresql or sqlite due close mysql querys)

### ISSUE THREAD

**Issue by mckaygerhard** [dsoares/Roundcube-Plugin-Lastlogin#1]
in the functions "lasst_log" and "write_log" does not use query with the RC api

and in any way the query are not SQL standar so this plugins that its a great feature does not are usefully wuith real DBMS like oracle, postgresql or sybase using odbc or pgpsql drivers

- i made yesterday some fine tune and table creation are not necesary if used a own made log file, that made more portable and better using egrep and shell scripts (roundcube have many shell scripts in bin directory)
-  **the id are innecesary in insert, due are autogenerated**
-  but using portability, the insert query can be modified, and the table creation for sqlite and postgres/oracle are easy..
-  the last_log function use a xtrange query i dont understand:
UNIX_TIMESTAMP(timestamp) ?????
real_ip:ip ???
in unix timestamp are always same (BSD, linux, Mac), guindows system must use posix or fuck'up so dont care! 
- and the return value 
$rows[] = $res;
are php 5.4+ so must be changed to complaint php 5.3 
array_push($rows, $res)

]I can made some pull request to fixed that important topis

**Comment by dsoares** [comment]
I've pushed to master some changes to make the SQL queries more SQL standard.

> i made yesterday some fine tune and table creation are not necesary if used a own made log file, that made more portable and better using egrep and shell scripts (roundcube have many shell scripts in bin directory)

I don't see the point of using a file, since there is already a userlogins log file created by Roundcube (see `program/include/rcmail.php`). I created the table because i need it for other scripts i implemented (very specific to my organization) that do a bunch of SQL queries.

UNIX_TIMESTAMP(field) returns the field value in seconds since '1970-01-01 00:00:00' UTC as an unsigned integer. I've translated it to sqlite and postgresql. You are welcome to contribute with those drivers initial setup scripts.

As i said in #2 `$rows[] = $res` is compatible with PHP 5.3.
